library/src/mixins/property/filtered_mixin_imports/remove_existing_imports_fy.py

```diff
@@ -0,0 +1,67 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+"""fy
+from typing import List
+
+
+property filtered_mixin_imports: List[str] using remove_existing_imports:
+    property mixin_imports
+    property parsed_fy_py_file
+"""
+
+from functools import cached_property
+
+from constants import IMPORT_REGEX
```

Comment:
If `IMPORT_REGEX` is not used elsewhere, then I would move it here and made it private to this file.